specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2021-04-01-preview/apimpolicyfragments.json

```diff
@@ -0,0 +1,343 @@
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version. Should be treated as opaque and used to make conditional HTTP requests.",
+                "type": "string"
+              }
+            }
+          },
+          "200": {
+            "description": "Policy fragment configuration of the Api Management service was successfully updated.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/PolicyFragmentContract"
+            },
+            "headers": {
+              "ETag": {
+                "description": "Current entity state version",
+                "type": "string"
+              }
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "PolicyFragment"
+        ],
+        "operationId": "PolicyFragment_Delete",
+        "description": "Deletes the policy fragment configuration of the Api Management Service.",
+        "x-ms-examples": {
+          "ApiManagementDeletePolicy": {
+            "$ref": "./examples/ApiManagementDeletePolicyFragment.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./apimanagement.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ServiceNameParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/PolicyFragmentIdParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/IfMatchRequiredParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "./apimanagement.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Policy fragment was successfully removed"
+          },
+          "204": {
+            "description": "Policy fragment successfully removed by previous request or does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./apimanagement.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ApiManagement/service/{serviceName}/policyFragments/{policyFragmentId}/references": {
+      "get": {
+        "tags": [
+          "PolicyFragment"
+        ],
+        "operationId": "PolicyFragment_GetReferences",
+        "description": "Lists resources that reference the policy fragment of the Api Management service.",
```

Comment:
It's really listing the policy resources that reference the specified policy fragment within the API management service (customer facing documentation)

Reply:
I updated the descriptions. Please review and comment if you think it should be changed again. Thanks.